tests/lib/Importer/SchemaImporterTest.php

```diff
@@ -25,172 +25,205 @@ class SchemaImporterTest extends TestCase
+                                ->setAutoincrement(true),
+                            new Column('main_id', Type::getType('integer')),
+                        ],
+                        [
+                            new Index('primary', ['id'], false, true),
+                        ],
+                        [
+                            new ForeignKeyConstraint(
+                                ['main_id'],
+                                'my_main_table',
+                                ['id'],
+                                'fk_my_secondary_table_id_main',
+                                ['onDelete' => 'CASCADE', 'onUpdate' => 'CASCADE']
+                            ),
+                        ]
+                    ),
+                ]
+            ),
+        ];
+
+        yield [
+            '04-nullable_field.yaml',
+            new Schema(
+                [
+                    new Table(
+                        'my_table',
+                        [
+                            (new Column('data', Type::getType('integer')))->setNotnull(false),
+                        ]
+                    ),
+                ]
+            ),
+        ];
+
+        yield [
+            '05-varchar_length.yaml',
+            new Schema(
+                [
+                    new Table(
+                        'my_table',
+                        [
+                            (new Column('name', Type::getType('string')))->setLength(64),
+                        ]
+                    ),
+                ]
+            ),
+        ];
+
+        yield [
+            '06-index.yaml',
+            new Schema(
+                [
+                    new Table(
+                        'my_table',
+                        [
+                            new Column('data1', Type::getType('integer')),
+                            new Column('data2', Type::getType('integer')),
+                            new Column('name', Type::getType('string')),
+                        ],
+                        [
+                            new Index('ix_simple', ['data1'], false, false),
+                            new Index('ix_composite', ['data1', 'data2'], false, false),
+                            new Index('ux_name', ['name'], true, false),
+                        ]
+                    ),
+                ]
+            ),
+        ];
+
+        yield [
+            '07-numeric-options.yaml',
+            new Schema(
+                [
+                    new Table(
+                        'my_table',
+                        [
+                            (new Column(
+                                'data',
+                                Type::getType('decimal')
+                            )
+                            )->setPrecision(19)->setScale(4),
+                        ]
+                    ),
+                ]
+            ),
+        ];
+
+        yield [
+            'short-index.yaml',
```

Comment:
I'd rather do `simple-index` to be consistent with the feature name (PR title).

Reply:
Fixed in 06c598de